Library/Formula/assimp.rb

```diff
@@ -21,7 +20,9 @@ class Assimp < Formula
   depends_on "boost" => [:recommended, :build]
 
   def install
-    system "cmake", ".", *std_cmake_args
+    args = std_cmake_args
+    args << "-DASSIMP_BUILD_TESTS=OFF"
```

Comment:
Building the tests downloads gtest, but the gtest repository has moved since this version (3.2) of assimp was released, so building the tests fails.
Building from HEAD doesn't have this problem. When the next assimp release happens, we can turn the tests back on and run the `unit` program that gets generated to test for success. (We wouldn't want to link it, though.)